presto-main/src/main/java/com/facebook/presto/server/ClusterStatsResource.java

```diff
@@ -141,7 +151,8 @@ public ClusterStats(
                 @JsonProperty("reservedMemory") double reservedMemory,
                 @JsonProperty("totalInputRows") long totalInputRows,
                 @JsonProperty("totalInputBytes") long totalInputBytes,
-                @JsonProperty("totalCpuTimeSecs") long totalCpuTimeSecs)
+                @JsonProperty("totalCpuTimeSecs") long totalCpuTimeSecs,
+                @JsonProperty("readyForQuery") boolean readyForQuery)
```

Comment:
better to call it: "isActive" (true/false) or "state": (ACTIVE/INACTIVE) or somewhat similar name. Cluster can accept query even without any workers, those queries will just be queued. So `readyForQuery` doesn't make sense here.

Reply:
updated to "isActive"